src/main/resources/templates/index.html

```diff
@@ -1,11 +1,8 @@
-<html>
-	<head>
-		<link rel="stylesheet" href="/css/styles.css" />
-	</head>
-	<body>
-		<div>Hiasdadsasssssssdsds</div>
-		<div>pre {{example}} post</div>
-		no
-		<pre>{{profiles}}</pre>
-	</body>
-</html>
+{% extends "common" %}
```

Comment:
Please comment and note if this is a directly accessible page, or another template file.